src/GeositeFramework/js/Plugin.js

```diff
@@ -475,7 +475,8 @@ require(['use!Geosite',
                     hasHelp: pluginObject.hasHelp,
                     hasCustomPrint: pluginObject.hasCustomPrint,
                     sizeClassName: getPluginSizeClassName(pluginObject.size),
-                    customWidth: getCustomPluginWidth(pluginObject.size, pluginObject.width)
+                    customWidth: getCustomPluginWidth(pluginObject.size, pluginObject.width),
+                    hideMinimizeButton: pluginObject.hideMinimizeButton
```

Comment:
Not sure it's worth changing but out of curiosity I checked and it seems like the minimize button will be hidden if `overrides.json` has any truthy value for `hideMinimizeButton`. Since it's an override that's probably the behavior we want -- a plugin developer would have to intentionally add the `hideMinimizeButton` key to overrides.json in the first place, which seems like a pretty good proxy for intent.

Reply:
Yeah, I think that's okay. I modeled the implementation in the launchpad from regional planning: https://github.com/CoastalResilienceNetwork/regional-planning/blob/master/main.js#L66